common/msp/localmsp.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package msp
+
+import (
+	"fmt"
+
+	"github.com/hyperledger/fabric-lib-go/bccsp/factory"
+	"github.com/hyperledger/fabric-x-orderer/config/bccsp"
+	"github.com/hyperledger/fabric/msp"
+)
+
+func BuildLocalMSP(localMSPDir string, localMSPID string, bccspConfig *bccsp.BCCSP) *msp.MSP {
```

Comment:
What if, instead of our `bccspConfig *bccsp.BCCSP` we would read the `factory.FactoryOpts` directly from the local config  yaml ?